README.md

```diff
@@ -102,7 +103,7 @@ render(<Picker />, mountNode);
 | renderExtraFooter | (mode) => React.Node |  | extra footer |
 | onSelect | Function(date: moment) |  | a callback function, can be executed when the selected time |
 | onPanelChange | Function(value: moment, mode) |  | callback when picker panel mode is changed |
-| onMouseDown | (evnet:React.MouseEventHandler<HTMLInputElement>) => void |  | callback when executed onMouseDown evnent |
+| onMouseDown | (event:React.MouseEventHandler<HTMLInputElement>) => void |  | callback when executed onMouseDown evnent |
```

Comment:
same as https://github.com/react-component/picker/pull/138/files#r545639435

Reply:
okay, will do

Reply:
after read the code, it is using ```FocusEventHandler``` instead of ```FocusEvent``` so the readme section is good i think?

Reply:
It is better to understand in this way than FocusEventHandler in the document.

Reply:
oh yes, it is